tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -10,13 +10,11 @@ Test Setup          fail test fast if required
 
 Force Tags          Admin    Local    Dev    AltersData
 
-
```

Comment:
Are these removed newlines here a result of you not having the latest `pipenv install` dependencies from root e.g. using an older version of the formatter?